Sources/OpenSSL/shim.h

```diff
@@ -196,4 +196,19 @@ static inline void EVP_CIPHER_CTX_free_wrapper(EVP_CIPHER_CTX *ctx) {
         #endif
 }
 
+// Converts the given string to BIGNUM
+static inline BIGNUM *stringToBigNum(const unsigned char *p_str, int len) {
+
+        int count = 0;
+        unsigned char str[len];
```

Comment:
Why do we need to copy `p_str` into another buffer? Couldn't we feed `p_str` straight to `BN_bin2bn`?

Reply:
Also, I don't see the need for the _len_ parameter...  _strlen()_ should be more than sufficient...